### PR TITLE
websocket issue cluster mode

### PR DESCRIPTION
**Pull request by shohamroditimemphis**

### CODE REVIEW

File: server/memphis_handlers_ws.go [hunk 1]
```diff
@@ -67,7 +67,7 @@ func memphisWSLoop(s *Server, subs map[string]memphisWSReqFiller, quitCh chan st
 		select {
 		case <-ticker.C:
 			for k, updateFiller := range subs {
-				replySubj := fmt.Sprintf(memphisWS_TemplSubj_Publish, k)
+				replySubj := fmt.Sprintf(memphisWS_TemplSubj_Publish, k+"_"+configuration.SERVER_NAME)
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_ws.go:70]: use . instead of _

File: server/memphis_handlers_ws.go [hunk 2]
```diff
@@ -130,6 +130,23 @@ func (s *Server) createWSRegistrationHandler(h *Handlers) simplifiedMsgHandler {
 		default:
 			s.Errorf("memphis websocket: invalid sub/unsub operation")
 		}
+		if configuration.SERVER_NAME == "" {
+			configuration.SERVER_NAME = "BROKER"
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_ws.go:134]: switch capital to non capital letters

File: ui_src/src/components/filter/index.js [hunk 3]
```diff
@@ -74,41 +76,49 @@ const Filter = ({ filterComponent, height }) => {
...
-
-                setTimeout(() => {
-                    state.socket?.publish(`$memphis_ws_subs.get_all_stations_data`, sc.encode('SUB'));
+                setTimeout(async () => {
+                    try {
+                        const rawBrokerName = await state.socket?.request(`$memphis_ws_subs.get_all_stations_data`, sc.encode('SUB'));
+                        const brokerName = JSON.parse(sc.decode(rawBrokerName._rdata))['name'];
+                        sub = state.socket?.subscribe(`$memphis_ws_pubs.get_all_stations_data.${brokerName}`);
+                    } catch (err) {
+                        console.log(`problem with request: ${err}`);
```
Comment (avrhamNeeman) [inline ui_src/src/components/filter/index.js:87]: We don't print to console the errors

File: ui_src/src/components/filter/index.js [hunk 4]
```diff
@@ -74,41 +76,49 @@ const Filter = ({ filterComponent, height }) => {
...
+                    }
+                    if (sub) {
+                        (async () => {
+                            for await (const msg of sub) {
+                                let data = jc.decode(msg.data);
+                                data.sort((a, b) => new Date(b.station.creation_date) - new Date(a.station.creation_date));
```
Comment (avrhamNeeman) [inline ui_src/src/components/filter/index.js:93]: data?.sort

**Review by idanasulinmemphis — APPROVED** [review]

**Pull request merged**